src/AppInstallerCommonCore/Public/winget/UserSettings.h

```diff
@@ -129,6 +131,8 @@ namespace AppInstaller::Settings
         SETTINGMAPPING_SPECIALIZATION(Setting::NetworkDOProgressTimeoutInSeconds, uint32_t, std::chrono::seconds, 60s, ".network.doProgressTimeoutInSeconds"sv);
         SETTINGMAPPING_SPECIALIZATION(Setting::InstallLocalePreference, std::vector<std::string>, std::vector<std::string>, {}, ".installBehavior.preferences.locale"sv);
         SETTINGMAPPING_SPECIALIZATION(Setting::InstallLocaleRequirement, std::vector<std::string>, std::vector<std::string>, {}, ".installBehavior.requirements.locale"sv);
+        SETTINGMAPPING_SPECIALIZATION(Setting::InstallLocation, std::vector<std::string>, std::vector<std::string>, {}, ".installLocation"sv);
+        SETTINGMAPPING_SPECIALIZATION(Setting::AddToPath, bool, bool, true, ".addToPath"sv);

```

Comment:
>addToPath [](http://example.com/codeflow?start=78&length=9)

nit: It feels to me this name is too broad

Reply:
Removed this and will address in a future PR